ovs/lib/helpers/generic/scrubber.py

```diff
@@ -394,7 +396,8 @@ def _update_vdisk_scrubbing_info():
             expires = now + 30
             data = {'job_id': self.job_id,
                     'expires': expires,
-                    'on_going': True}
+                    'on_going': True,
```

Comment:
ongoing is written in 1 word. Is changing the variable name a hassle? backwards compatible?